src/paymentservice/index.js

```diff
@@ -69,4 +69,15 @@ const PROTO_PATH = path.join(__dirname, '/proto/');
 
 const server = new HipsterShopServer(PROTO_PATH, PORT);
 
+const pino = require('pino');
+const logger = pino({
+  name: 'paymentservice-server',
+  messageKey: 'message',
+  formatters: {
+    level (logLevelString, logLevelNum) {
+      return { severity: logLevelString }
+    }
+  }
+});
+
```

Comment:
**Issue:** The paymentservice is failing deployment:
```
... deployment/paymentservice: container server terminated with exit code 1 ...
... /usr/src/app/index.js:21
... logger.info("Profiler disabled.")
... ReferenceError: Cannot access 'logger' before initialization
```
You will need to move the chunk that initializes the `logger` object to the top of the file (after `'use strict';`).

Reply:
Sorry about that @NimJay, actually I already initialized logger at the top of the file and tested it out locally but I forgot to commit those changes to my branch.